style/care.css

```
@@ -0,0 +1,69 @@
+
+.intro {
+    margin-top: 1em;
+    height: 275px;
+    display: flex;
+    width:100%;  
+}
+
+#summary {
+    width:55%;
+    background-color: #436105;
+    background-image: url(../assets/intro.jpg);
+    background-size: cover;
+
+
+}
+
+#summary p{
+    display: block;
+    height: 10px;
+    width: 400px;
+    margin-left: auto;
+    margin-right: 1em;
+    background-color: white;
+    padding: 1em;
+}
+
+#table-of-contents{
+    background-color: white;
+    padding-left: 20px;
+    width: 50%;
+}
+
+#table-of-contents h2 {
+    margin: 0;
+}
+
+.content {
+    margin-left: 1em;
+    /* background-color: red; */
+    max-width: 1000px;
```

```suggestion
    margin-left: 1em;
    max-width: 1000px;
```

Remove commented out code